libs/design-system/src/lib/Pages/shared/GrayPage.tsx

```diff
@@ -0,0 +1,15 @@
+import clsx from 'clsx';
+
+export default function GrayPage({
```

Comment:
What do you think about renaming this to `PageBackground`?

Reply:
Sure, just renamed it!